Userland/Applications/FontEditor/MainWidget.cpp

```diff
@@ -471,7 +471,7 @@ void MainWidget::update_action_text()
         TRY(builder.try_append(prefix));
         if (suffix.has_value()) {
             TRY(builder.try_append(' '));
-            TRY(builder.try_append(suffix.value()));
+            TRY(builder.try_append(suffix.value().release_value_but_fixme_should_propagate_errors()));
```

Comment:
This function already can propagate errors:
```suggestion
            TRY(builder.try_append(TRY(suffix.value())));
```